node/pkg/proto/publicrpc/v1/publicrpc.pb.go

```diff
@@ -25,24 +25,24 @@ const (
 type ChainID int32
 
 const (
-	ChainID_CHAIN_ID_UNSPECIFIED      ChainID = 0
-	ChainID_CHAIN_ID_SOLANA           ChainID = 1
-	ChainID_CHAIN_ID_ETHEREUM         ChainID = 2
-	ChainID_CHAIN_ID_TERRA            ChainID = 3
-	ChainID_CHAIN_ID_BSC              ChainID = 4
-	ChainID_CHAIN_ID_POLYGON          ChainID = 5
-	ChainID_CHAIN_ID_AVALANCHE        ChainID = 6
-	ChainID_CHAIN_ID_OASIS            ChainID = 7
-	ChainID_CHAIN_ID_ALGORAND         ChainID = 8
-	ChainID_CHAIN_ID_AURORA           ChainID = 9
-	ChainID_CHAIN_ID_FANTOM           ChainID = 10
-	ChainID_CHAIN_ID_KARURA           ChainID = 11
-	ChainID_CHAIN_ID_ACALA            ChainID = 12
-	ChainID_CHAIN_ID_KLAYTN           ChainID = 13
-	ChainID_CHAIN_ID_CELO             ChainID = 14
-	ChainID_CHAIN_ID_NEAR             ChainID = 15
-	ChainID_CHAIN_ID_MOONBEAM         ChainID = 16
-	ChainID_CHAIN_ID_NEON             ChainID = 17
+	ChainID_CHAIN_ID_UNSPECIFIED ChainID = 0
+	ChainID_CHAIN_ID_SOLANA      ChainID = 1
+	ChainID_CHAIN_ID_ETHEREUM    ChainID = 2
+	ChainID_CHAIN_ID_TERRA       ChainID = 3
+	ChainID_CHAIN_ID_BSC         ChainID = 4
+	ChainID_CHAIN_ID_POLYGON     ChainID = 5
+	ChainID_CHAIN_ID_AVALANCHE   ChainID = 6
+	ChainID_CHAIN_ID_OASIS       ChainID = 7
+	ChainID_CHAIN_ID_ALGORAND    ChainID = 8
+	ChainID_CHAIN_ID_AURORA      ChainID = 9
+	ChainID_CHAIN_ID_FANTOM      ChainID = 10
+	ChainID_CHAIN_ID_KARURA      ChainID = 11
+	ChainID_CHAIN_ID_ACALA       ChainID = 12
+	ChainID_CHAIN_ID_KLAYTN      ChainID = 13
+	ChainID_CHAIN_ID_CELO        ChainID = 14
+	ChainID_CHAIN_ID_NEAR        ChainID = 15
+	ChainID_CHAIN_ID_MOONBEAM    ChainID = 16
+	// OBSOLETE: CHAIN_ID_NEON = 17;
```

Comment:
I was thinking when I first pulled this PR up to check to see if you did this. Nice.

I'd be half tempted to make a unit test to ensure that ChainID 17 is never used. Maybe use the machinery that converts the chainId to a string and if it returns anything then fail the test or something.